test/cypress/support/exception-handler.js

```diff
@@ -0,0 +1,2 @@
+// returning false here prevents Cypress from failing the test
+Cypress.on('uncaught:exception', () => false);
```

Comment:
With this Cypress will not fail if there is an error in the console. Maybe we should reconsider it.

Reply:
we have some errors atm in console so we need to keep this as it is for now until we fix all errors

Reply:
sounds good.